docker/ci/x86_64/Dockerfile

```diff
@@ -1,25 +1,22 @@
-
-# must be built from /dist directory
-
-FROM ubuntu:20.04 as prep
-LABEL MAINTAINER="https://discord.gg/Uz29ny"
-
-RUN apt-get update && \
-    apt-get -y install curl xz-utils && \
-    apt-get autoclean -y && \
-    rm -rf /var/lib/apt/lists/*
+#FROM --platform=$BUILDPLATFORM ubuntu:20.04  AS prep
+FROM --platform=$BUILDPLATFORM alpine  AS prep
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+RUN apk add --no-cache curl xz
 WORKDIR /
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-
-RUN curl --http1.1 -o /ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz && \
+COPY stash-*  /
+RUN if [ "$TARGETPLATFORM" = "linux/arm/v6" ]; then FFMPEG_URL=https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-armel-static.tar.xz;BIN=stash-pi; elif [ "$TARGETPLATFORM" = "linux/arm/v7" ]; then FFMPEG_URL=https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-armhf-static.tar.xz;BIN=stash-linux-arm32v7; elif [ "$TARGETPLATFORM" = "linux/arm64" ]; then FFMPEG_URL=https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-arm64-static.tar.xz;BIN=stash-linux-arm64v8;elif [ "$TARGETPLATFORM" = "linux/amd64" ]; then FFMPEG_URL=https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz;BIN=stash-linux;  fi; \
```

Comment:
You're splitting this command onto the lines below, so split this if/elif line into a more readable format.